src/bin/CMakeLists.txt

```diff
@@ -33,3 +33,21 @@ else()
     SET_TARGET_PROPERTIES(msquic
         PROPERTIES LINK_FLAGS "-Wl,--version-script=\"${CMAKE_CURRENT_SOURCE_DIR}/linux/exports.txt\"")
 endif()
+
+target_include_directories(msquic PUBLIC
+                            $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../inc>
+                            $<INSTALL_INTERFACE:${include_dest}>)
```

Comment:
Our style so far has been to put new lines just 4 spaces after the start of the first line.